tests/test-pow.c

```diff
@@ -49,31 +81,58 @@ int main()
 
     int mwm = 14;
 
-    /* test implementation of the PoW FPGA accelerator with mwm = 14 */
-    initializeImplContext(&PoWFPGAAccel_Context);
-    void *pow_ctx =
-        getPoWContext(&PoWFPGAAccel_Context, (int8_t *) trytes, mwm);
-    assert(pow_ctx);
-    doThePoW(&PoWFPGAAccel_Context, pow_ctx);
-    int8_t *ret_trytes = getPoWResult(&PoWFPGAAccel_Context, pow_ctx);
-    assert(ret_trytes);
-    freePoWContext(&PoWFPGAAccel_Context, pow_ctx);
-
-    Trytes_t *trytes_t = initTrytes(ret_trytes, 2673);
-    assert(trytes_t);
-    Trytes_t *hash_trytes = hashTrytes(trytes_t);
-    assert(hash_trytes);
-
-    /* Validation */
-    Trits_t *ret_trits = trits_from_trytes(hash_trytes);
-    for (int i = 243 - 1; i >= 243 - mwm; i--) {
-        assert(ret_trits->data[i] == 0);
-    }
+    ImplContext ImplContextArr[ ] = {
+        #if defined(ENABLE_AVX)
```

Comment:
Don't indent `#if` and `#endif` statements. Always leave them without indentation (no leading space).